shell.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+
+sys.path.insert(0, os.path.dirname(__file__))
+
+
+from yolapy.services import Yola
+from tests.test_integration.test_settings import url, auth
```

Comment:
This doesn't seem like the main program of yolapy. I don't think yolapy has a main program/purpose. This is a developer tool. I can't think of an alternative suggestion that isn't root though...